examples/analysis-botanalytics/README.md

```diff
@@ -1,12 +1,12 @@
-# With botanalytics
+# Analysis botanalytics
```

Comment:
Would it be better use title like: 
# Analysis: Botanalytics
?

Reply:
Sure